apps/wistia-videos/src/functions/getVideos.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  Project,
+  ProjectReduced,
+  WistiaError,
+  WistiaVideo,
+  WistiaProject,
+} from '../components/helpers/types';
+
+const VIDEOS_PER_PAGE = 100;
+
+export const fetchProjects = async (bearerToken: string): Promise<WistiaProject[]> => {
+  const projects = await (
+    await fetch(`https://api.wistia.com/v1/projects.json`, {
+      headers: {
+        Authorization: `Bearer ${bearerToken}`,
+      },
+    })
+  ).json();
+  if (projects.error) {
+    throw new WistiaError({ message: projects.error, code: projects.code });
+  } else {
+    const reducedProjects = projects.map(({ id, hashedId, name }: ProjectReduced) => {
+      return { id, hashedId, name };
+    });
+
+    return reducedProjects;
+  }
+};
+
+const getTotalPages = (count: number) => {
+  return Math.ceil(count / VIDEOS_PER_PAGE);
+};
+
+export const fetchVideos = async (
+  projectIds: string[],
+  bearerToken: string
+): Promise<WistiaVideo[]> => {
+  const mappedProjects = await Promise.all(
+    projectIds.map(async (id: string) => {
+      const project: WistiaProject = await (
+        await fetch(`https://api.wistia.com/v1/projects/${id}.json`, {
+          headers: {
+            Authorization: `Bearer ${bearerToken}`,
+          },
+        })
+      ).json();
+      let videos: WistiaVideo[] = project.medias;
```

Comment:
move this variable down to line 68 and make this a const

Reply:
I actually just deleted this var and concatted the new array to the existing `project.medias`